Tests/StreamChatTests/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -614,6 +614,153 @@ final class ChannelDTO_Tests: XCTestCase {
                 .map(\.user.id)
         )
     }
+    
+    func test_lastMessageFromCurrentUser() throws {
+        let user: UserPayload = dummyCurrentUser
+        let channelId: ChannelId = .unique
+        let message1 = MessagePayload(
```

Comment:
Feel free to use `mock` and pass just the fields you're interested in:
```suggestion
        let message1: MessagePayload = .mock()
```

Reply:
Personally I consider that the initializer for the `MessagePayload` is more clear than the `.dummy()` that you suggest me to use because that dummy has different parameter names that the ones expected by the init and also has some logic and validations for some of the properties.